### ISSUE TITLE
Can't resolve SVGID after upgrading to 1.7

### ISSUE THREAD

**Issue by JohannesRudolph** [angular/angular-cli#9665]
### Versions
```
λ  ng --version

    _                      _                 ____ _     ___
   / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
  / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
 / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
/_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
               |___/

Angular CLI: 1.7.0
Node: 6.12.3
OS: win32 x64
Angular: 5.2.4
... common, compiler, compiler-cli, core, forms, http
... platform-browser, platform-browser-dynamic, router

@angular/cli: 1.7.0
@angular-devkit/build-optimizer: 0.3.1
@angular-devkit/core: 0.3.1
@angular-devkit/schematics: 0.3.1
@ngtools/json-schema: 1.2.0
@ngtools/webpack: 1.10.0
@schematics/angular: 0.3.1
@schematics/package-update: 0.3.1
typescript: 2.4.2
webpack: 3.11.0
```

### Repro steps
Add a css class to styles.css that references a svg resource

### Observed behavior
Build fails with lots of very similar errors (only the number after `SVGID_` changes): 
```
ERROR in ./src/styles.scss
(Emitted value instead of an instance of Error) CssSyntaxError: C:\dev\mc\meshfed\panel\src\styles.scss:5947:983: Can't resolve '%23SVGID_2_' in 'C:\dev\mc\meshfed\panel\src'

  5945 |
  5946 | #preloader {
> 5947 |   background-image: url("assets/core/texture.svg");
       |                                                    ^
  5948 |   background-repeat: no-repeat;
  5949 |   background-size: cover;
```

### Desired behavior
No errors during builds

### Mention any other details that might be useful (optional)
Build worked fine before on ng-cli v1.6.7

**Comment by flmg** [comment]
I have the same error after upgrading to 1.7.0. **Only when building in production** (`ng build --env=prod`) I get the following errors in one of my dependencies:

    ERROR in ./node_modules/raw-loader!./node_modules/postcss-loader/lib??embedded!./node_modules/less-loader/dist/cjs.js??ref--9-3!./node_modules/flag-icon-css/less/flag-icon.less
    (Emitted value instead of an instance of Error) CssSyntaxError: 
    /Users/.../node_modules/flag-icon-css/less/flag-icon-
    base.less:25:6: Can't resolve '%23d' in '/Users/.../node_modules/flag-icon-css/less'

      23 |     background-image: ~"url(@{flag-icon-css-path}@{flag-icon-rect-path}/@{country}.svg)";
      24 |     &.flag-icon-squared {
    > 25 |       background-image: ~"url(@{flag-icon-css-path}@{flag-icon-square-path}/@{country}.svg)";
         |      ^
      26 |     }
      27 |   }

It works fine with `1.6.8`.

**Comment by abdulkareemnalband** [comment]
I'm getting same error but like this:

```
Can't resolve '%23mask-2'
```

instead of 
```
Can't resolve '%23SVGID_2_'
```

**Comment by clydin** [comment]
Duplicate of #9642; fixed via #9660; and will be in the next patch release.
